### PR TITLE
Fix Skeleton Bounding Box Timing issue

### PR DESCRIPTION
**Pull request by sebavan**
https://forum.babylonjs.com/t/bounding-box-does-not-compute-correctly/38448/4

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13561/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13561/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13561/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13561/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13561/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13561/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13561/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13561/merge#BCU1XR#0

File: packages/dev/core/src/Bones/skeleton.ts [hunk 1]
```diff
@@ -189,6 +189,10 @@ export class Skeleton implements IAnimatable {
             this.prepare();
         }
 
+        if (this._isDirty) {
```
Comment (Popov72) [inline packages/dev/core/src/Bones/skeleton.ts:192]: nit: you could have merged this `if` with the previous one

**Review by Popov72 — APPROVED** [review]

File: packages/dev/core/src/Bones/skeleton.ts (hunk 1 above)
Comment (sebavan) [reply packages/dev/core/src/Bones/skeleton.ts]: yes not sure why I did not...

**Pull request merged**